lib/routes/admin-api/user.js

```diff
@@ -3,17 +3,21 @@
 const Controller = require('../controller');
 
 class UserController extends Controller {
-    constructor() {
-        super();
+    constructor(perms) {
+        super(perms);
         this.get('/', this.getUser);
         this.get('/logout', this.logout);
     }
 
     getUser(req, res) {
         if (req.user) {
+            const user = Object.assign({}, req.user);
+            if (!this.extendedPermissions) {
+                delete user.permissions;
+            }
```

Comment:
@ivarconr my solution

Reply:
So if the user object does not have any permissions it safe to assume that permissions are disabled?

Reply:
good question, either we trust the middleware from the user to always set permissions in the extendedPermissions case, or we have to do an additional check.

Reply:
Let's start with this, if needed to make the UI better we can always introduce some extra options.